src/iproute2_sysrepo.c

```diff
@@ -299,30 +299,35 @@ struct filter_util *get_filter_kind(const char *str)
 	return q;
 }
 
-static int do_cmd(const char *argv0, int argc, char **argv)
+static int do_cmd(int argc, char **argv)
```

Comment:
could argv be const'ified too ?

Reply:
tried it set argv to const, but got so some warrnings, for callers, i need to cast the caller to resovle this, let's keep it as it is for now, what do you think ?